tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+add_executable(sof-fuzzer
+	main.c
+	qemu-bridge.c
+	../testbench/topology.c
+	platform/byt-host.c
+)
+
+target_link_libraries(sof-fuzzer PRIVATE
+	"-lglib-2.0 -lrt"
+)
+
+target_compile_definitions(sof-fuzzer PRIVATE -DBUILD_FUZZER=1)
+
+find_package(PkgConfig REQUIRED)
+pkg_search_module(GLIB REQUIRED glib-2.0)
+target_include_directories(sof-fuzzer PRIVATE ${GLIB_INCLUDE_DIRS})
+target_link_libraries(sof-fuzzer INTERFACE ${GLIB_LDFLAGS})
+
+target_include_directories(sof-fuzzer PRIVATE
+	"/usr/include/glib-2.0"
+	"/usr/lib/x86_64-linux-gnu/glib-2.0/include"
```

Comment:
These `"/usr/include/glib-2.0"` & `"/usr/lib/x86_64-linux-gnu/glib-2.0/include"` should be already covered in: `target_include_directories(sof-fuzzer PRIVATE ${GLIB_INCLUDE_DIRS})`

Reply:
oops yes. Thanks, removed now.